cuda_core/tests/test_program.py

```diff
@@ -7,20 +7,12 @@
 # is strictly prohibited.
 
 import pytest
+from conftest import can_load_generated_ptx
```

Comment:
My understanding is that we don't need explicit imports for things from conftest?

Reply:
I believe this only aplies to fixtures. helper functions still need to be imported. This is what my research and tests have shown.

Reply:
I could add a new file called utils, or helpers and import that instead. There is mixed opinions on best practice online

Reply:
> This is what my research and tests have shown.

I was only relying on chatgpt before. It told me the imports are not needed. If that's not true: keeping it simple seems best to me, unless you anticipate that we're accumulating many helper functions. I.e. just keep what you have?

Reply:
yeah, it told me that too, then changed its mind. Keeping it simple was my idea as well, with only 1 helper function, it seems premature to split it into a new file, but perhaps down the line it will make sense

Reply:
yeah we can move it to `tests/utils` next time we touch this file